src/swap/db.rs

```diff
@@ -0,0 +1,9 @@
+#[async_trait::async_trait]
+pub trait Load<T>: Send + Sync + 'static {
+    async fn load(&self, swap_id: u8) -> anyhow::Result<Option<T>>;
```

Comment:
📣 We need a real swap id here, not a `u8`.

Reply:
Yep, yep. I wasn't sure if I should introduce it with this PR as I don't actually have one when I start the swap. I expected to modify these functions once the concept is in place (with the introduction of the database, I guess).